kde-frameworks/purpose-common/purpose-common-9999.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_I18N="false" # translations don't collide
+ECM_HANDBOOK="false"
+KDE_ORG_NAME="${PN/-common/}"
+inherit ecm-common frameworks.kde.org
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS=""
+IUSE="webengine"
+
+RDEPEND="
+	!<kde-frameworks/purpose-5.116.0-r2:5
+	!<kde-frameworks/purpose-6.5.0-r1:6
+"
+BDEPEND="webengine? ( kde-apps/kaccounts-integration:* )"
+
+ECM_INSTALL_ICONS=(
+	src/plugins/reviewboard/icons/16-apps-reviewboard-purpose6.png:\${KDE_INSTALL_ICONDIR}
+	src/plugins/reviewboard/icons/128-apps-reviewboard-purpose6.png:\${KDE_INSTALL_ICONDIR}
+	src/plugins/phabricator/icons/16-apps-phabricator-purpose6.png:\${KDE_INSTALL_ICONDIR}
+	src/plugins/phabricator/icons/128-apps-phabricator-purpose6.png:\${KDE_INSTALL_ICONDIR}
+)
+
+ecm-common_inject_heredoc() {
+	if use webengine; then
```

Comment:
If you're using `<<-`, it'll strip leading whitespace, so you can indent the block below to look prettier.